fastkml/data.py

```diff
@@ -1,20 +1,32 @@
+from typing import Dict
 from typing import List
 from typing import Optional
 from typing import Tuple
 from typing import Union
 
 from typing_extensions import TypedDict
 
-import fastkml.config as config
 from fastkml.base import _BaseObject
 from fastkml.base import _XMLObject
+from fastkml.config import etree  # type: ignore[attr-defined]
```

Comment:
Please revert this change, otherwise the user is not able to change the etree implementation to use.

Reply:
Was trying to solve for `fastkml/data.py:148: error: Module has no attribute "etree"  [attr-defined]` - any suggestions for that?

Reply:
Those happen everytime we have a `config.etree` mention:
```
fastkml/data.py:148: error: Module has no attribute "etree"  [attr-defined]
fastkml/data.py:152: error: Module has no attribute "etree"  [attr-defined]
fastkml/data.py:178: error: Module has no attribute "etree"  [attr-defined]
fastkml/data.py:181: error: Module has no attribute "etree"  [attr-defined]
fastkml/data.py:302: error: Module has no attribute "etree"  [attr-defined]
```
I.e.:
`sf = config.etree.SubElement(element, f"{self.ns}SimpleField")`

Reply:
Reverting this will also make the failing tests pass